src/main/java/org/kainos/ea/db/EmployeeDao.java

```diff
@@ -38,4 +37,60 @@ public List<Employee> getAllEmployees() throws SQLException {
 
         return employeeList;
     }
-}
+
+    public int createNewEmployee(EmployeeRequest employeeRequest) throws SQLException, DeliveryEmployeeCouldNotBeCreatedException {
+
+        Connection c = databaseConnector.getConnection();
+
+        String insertStatement = "INSERT INTO employee (f_name, l_name, salary, bank_acc_num, ni_num) " +
+                "VALUES (?, ?, ?, ?, ?)";
+
+        PreparedStatement st = c.prepareStatement(insertStatement, Statement.RETURN_GENERATED_KEYS);
+
+        st.setString(1, employeeRequest.getF_name());
+        st.setString(2, employeeRequest.getL_name());
+        st.setDouble(3, employeeRequest.getSalary());
+        st.setString(4, employeeRequest.getBank_acc_num());
+        st.setString(5, employeeRequest.getNi_num());
+
+        st.executeUpdate();
+
+        ResultSet rs = st.getGeneratedKeys();
+
+        if (rs.next()) {
+
+            System.out.println("Adding New Employee Delivery.");
+
+            // Add New Create ID To Employee_delivery Table
+            int newlyCreatedIDEmployeeTable = rs.getInt(1);
+            rs = null;
```

Comment:
Should this be reset to `null`, or would it be good to use a second `ResultSet`?

Reply:
Fixed Now.